File: 백준/Bronze/11943. 파일 옮기기/파일 옮기기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int A = sc.nextInt();
        int B = sc.nextInt();
        int C = sc.nextInt();
        int D = sc.nextInt();
        int first = A + D;
        int second = C + B;
        if(first > second){
            System.out.println(second);
        }
        else{
            System.out.println(first);
        }
    }

}

File: 백준/Bronze/11943. 파일 옮기기/파일 옮기기.cc
#include <iostream>
#include <string>
#include <algorithm>

using namespace std;

int main(){ ios_base::sync_with_stdio(false); cin.tie(NULL);
    
    int A, B, C, D;
    cin >> A >> B >> C >> D;
    int first, second;
    first = A + D;
    second = C + B;
    if(first > second){
        cout << second;
    }
    else{
        cout << first;
    }
}